diff --git a/build/modules/vc/data-integrity/W3cJsonLdCredentialService.js b/build/modules/vc/data-integrity/W3cJsonLdCredentialService.js
index c833d0108e8852127afac1c87cffca0aba1268dc..e3391b11fd9b7b7fe85b027eb68ea110e4324499 100644
--- a/build/modules/vc/data-integrity/W3cJsonLdCredentialService.js
+++ b/build/modules/vc/data-integrity/W3cJsonLdCredentialService.js
@@ -179,7 +179,7 @@ let W3cJsonLdCredentialService = class W3cJsonLdCredentialService {
             domain: options.domain,
             documentLoader: this.w3cCredentialsModuleConfig.documentLoader(agentContext),
         });
-        return utils_1.JsonTransformer.fromJSON(result, W3cJsonLdVerifiablePresentation_1.W3cJsonLdVerifiablePresentation);
+        return utils_1.JsonTransformer.fromJSON(result, W3cJsonLdVerifiablePresentation_1.W3cJsonLdVerifiablePresentation, { validate: !(result.verifiableCredential === undefined) });
     }
     /**
      * Verifies a presentation including the credentials it includes
